Ci failing: 

~~~sh
[ 12%] Building CXX object kratos/CMakeFiles/KratosCore.dir/Unity/unity_5_cxx.cxx.o
In file included from /__w/Kratos/Kratos/build/FullDebug/kratos/CMakeFiles/KratosCore.dir/Unity/unity_5_cxx.cxx:61:
/__w/Kratos/Kratos/kratos/input_output/stl_io.cpp:94:31: error: lambda capture 'this' is not used [-Werror,-Wunused-lambda-capture]
        create_entity_func = [this](
                              ^~~~
[ 12%] Building CXX object kratos/CMakeFiles/KratosCore.dir/Unity/unity_4_cxx.cxx.o
~~~

--- a/kratos/input_output/stl_io.cpp
+++ b/kratos/input_output/stl_io.cpp
@@ -91,7 +91,7 @@ void StlIO::ReadModelPart(ModelPart & rThisModelPart)
     std::function<void(ModelPart&, NodesArrayType&)> create_entity_func;
     const std::string new_entity_type = mParameters["new_entity_type"].GetString();
     if (new_entity_type == "geometry") {
-        create_entity_func = [this](
+        create_entity_func = [](
             ModelPart& rThisModelPart,
             NodesArrayType& rIndexes) {
                 rThisModelPart.CreateNewGeometry("Triangle3D3", rIndexes);

